actions/tahoe-lafs

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/python3
+# -*- mode: python -*-
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Configuration helper for Tahoe-LAFS.
+"""
+
+import argparse
+import grp
+import json
+import pwd
+import shutil
+import subprocess
+
+import os
+import ruamel.yaml
+
+from plinth import action_utils
+from plinth.modules.tahoe import (
+    introducer_name,
+    introducers_file,
+    storage_node_name,
+    tahoe_home,
+    introducer_furl_file, restart_storage_node)
+from plinth.utils import YAMLFile
+
+domain_name_file = os.path.join(tahoe_home, 'domain_name')
+
+
+def parse_arguments():
+    """Return parsed command line arguments as dictionary."""
+    parser = argparse.ArgumentParser()
+    subparsers = parser.add_subparsers(dest='subcommand', help='Sub command')
+
+    subparsers.add_parser('enable', help='Enable Tahoe-LAFS')
+    subparsers.add_parser('disable', help='Disable Tahoe-LAFS')
+    setup = subparsers.add_parser('setup',
+                                  help='Set domain name for Tahoe-LAFS')
+    setup.add_argument('--domain-name',
+                       help='The domain name to be used by Tahoe-LAFS')
+    subparsers.add_parser('create-introducer',
+                          help='Create and start the introducer node')
+    subparsers.add_parser('create-storage-node',
+                          help='Create and start the storage node')
+    subparsers.add_parser('autostart',
+                          help="Automatically start all introducers and "
+                               "storage nodes on system startup")
+    intro_parser_add = subparsers.add_parser('add-introducer',
```

Comment:
If you move 'add-introducer' down to the next line, the help text can take up a wider space, and be easier to read.